endpoints.md

```diff
@@ -0,0 +1,108 @@
+## Claim Post
+
+Url: /claim
+
+Method: Any
```

Comment:
Because we're sending JSON, all of these endpoints actually require a POST request.